R/produce_source_extract_tests.R

```diff
@@ -28,13 +29,19 @@ produce_source_extract_tests <- function(data,
                                          max_min_vars = c(
                                            "record_keydate1", "record_keydate2",
                                            "cost_total_net", "yearstay"
-                                         )) {
+                                         ),
+                                         add_hscp_count = TRUE) {
```

Comment:
`add_hscp_count` This needs updating in the documentation, and where it's used in `R/process_tests_outpatients.R`